src/standalone/Selector/MultiSelectEntry.tsx

```diff
@@ -10,7 +10,7 @@ import {
 import { SmallIconButton, SmallListItem, SmallListItemIcon } from "../Small";
 import { MultiSelectorData } from "./MultiSelect";
 import { ClassNameMap } from "@material-ui/styles/withStyles";
-import { RemoveIcon } from "../Icons";
```

Comment:
why did you change the icon?

Reply:
Same icon available in material UI icons. So, replaced with it.

Reply:
then delete the icon from components-care

Reply:
@TQjona , Removed from CC.